identity/res/layout/camera_permission_denied_fragment.xml

```diff
@@ -6,8 +6,9 @@
     tools:context=".navigation.CameraPermissionDeniedFragment">
 
     <TextView
+        android:id="@+id/info"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:text="Hello" />
+        android:text="CameraPermissionDenied" />
```

Comment:
should this be a `@string`?

Reply:
Yeah this is just a place holder now, will replace them when we are actually working on the fragment.